Tools/clinic/clinic.py

```diff
@@ -3978,9 +3981,15 @@ def add_c_return_converter(f, name=None):
 
 
 class CReturnConverterAutoRegister(type):
-    def __init__(cls, name, bases, classdict):
+    def __init__(
+            cls: ReturnConverterType,
+            name: str,
+            bases: tuple[type, ...],
+            classdict: dict[Any, Any]
```

Comment:
It's true that, strictly speaking, it's _possible_ to pass a dictionary with non-`str` keys in here, but it's almost certainly not what you want ;)

```pycon
>>> t = type('t', (), {'foo': 'bar', 3: 3})
>>> t
<class '__main__.t'>
>>> t.foo
'bar'
>>> t.3
  File "<stdin>", line 1
    t.3
     ^^
SyntaxError: invalid syntax
```

```suggestion
            classdict: dict[str, Any]
```

Reply:
Thanks! I'm not sure why I put `Any, Any` there; oh, well.